pkg/lib/serverTime.js

```diff
@@ -258,42 +267,21 @@ export function ServerTime() {
         return status;
     };
 
-    const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
-                                                { superuser: "try" });
+    function get_timesync_backend() {
+        return python.spawn(get_timesync_backend_py, [], { superuser: "try", err: "message" })
+                .then(data => data.trim());
+    }
+
+    function get_custom_ntp_timesyncd() {
+        const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
+                                                    { superuser: "try" });
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18298-20230207-124146-95ee9c44-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)